### DISCUSSION TITLE
Jam sync with the Gopro Mic adapter

### DISCUSSION THREAD

**Discussion by joeybassraft** [gopro/labs#1038] [General]
I am having trouble jam syncing my hero 11 black, is it because I'm using the Gopro mic adapter instead of the Media Mod? I am trying to jam sync it to my Rode wireless pro.
Thanks

**Comment by dnewman-gpsw** [comment]
I have used a Tenacle Sync via the old mac adaptor and it worked okay.  What is it doing?

**Comment by joeybassraft** [comment]
When I have the Wireless Pro on jam sync, the gopro accepts the input
audio, but I can't get it to display the green timecode on the screen, not
sure what to do?
I'm using the latest labs firmware and can use the QR code to set the
timecode. Any suggestions?
Thanks

**Reply by dnewman-gpsw** [reply]
There is a patch needed for H11 and H12, until any future releases.  It was fixed on H10 with *RLTC=1, but for now you use a boot command like !MBOOT="!Lbt"!SAVEbt=$PMSG="$pdB SPL"!N!R to turn audio processing on.

**Reply by joeybassraft** [reply]
That works, Thanks!

**Reply by jshawn-ai** [reply]
> There is a patch needed for H11 and H12, until any future releases. It was fixed on H10 with *RLTC=1, but for now you use a boot command like !MBOOT="!Lbt"!SAVEbt=$PMSG="$pdB SPL"!N!R to turn audio processing on.

apologies, I'm new to this and did this once a while back with a hero 11 and am now trying Jam sync with a hero 11 and hero 12 with no luck. so this command line is to be broken up into a series of custom qr codes for the cameras, is that correct?

**Comment by joeybassraft** [comment]
Yes, I copied the whole command and pasted it in the "Custom mode" box
below the QR code

**Reply by jshawn-ai** [reply]
Thank you for your help! It worked!
